3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1777,43 +1777,52 @@ L'elemento implementato deve essere approvato durante il processo di verifica (@
 _Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.4.8_
 
 === Scopo
+
 Il processo di integrazione ha lo scopo di combinare iterativamente elementi software al fine di ottenere un prodotto che soddisfi i requisiti rispettando l'architettura e il design definiti.
 È quindi necessario il coordinamento con il processo di definizione dell'architettura (@processo_definizione_architettura) e il processo di design (@processo_design).
 
 L'integrazione del sistema software avviene automaticamente mediante strumenti che permettano la Continuous Integration.
 
-
 === Risultati
+
 Come risultato della corretta implementazione del processo di integrazione:
+
 - viene integrato il nuovo elemento software con il prodotto principale;
 - vengono eseguiti automaticamente i test atti a garantire il corretto funzionamento del prodotto a seguito dell'integrazione;
 - vengono identificati i risultati ed eventuali anomalie;
 - i risultati dei test vengono registrati e possono essere visualizzati su GitHub;
 - gli elementi correttamente integrati vengono tracciati e possono essere visualizzati su GitHub.
 
 === Attività
+
 ==== Pianificazione della strategia di integrazione
 
-  La strategia di integrazione definita dal gruppo si basa sull'adozione delle pratiche "Continuous Integration" e "Continuous Deployment", al fine di garantire un frequente allineamento degli ambienti di lavoro tra i membri e avere costantemente la versione più aggiornata del prodotto disponibile e funzionante.
+  La strategia di integrazione definita dal gruppo si basa sull'adozione delle pratiche "Continuous Integration" e "Continuous Deployment", al fine di garantire un frequente allineamento degli ambienti di lavoro tra i membri ed avere costantemente la versione più aggiornata del prodotto disponibile e funzionante.
```

Comment:
Cambierei in

"Il gruppo deve basare la propria strategia di integrazione sulle pratiche di _continuous integration_ e _continuous deployment_, al fine di permettere il frequente allineamento degli ambienti di lavoro tra i membri e disporre sempre della versione più recente del prodotto."